Userland/Applications/Browser/Tab.cpp

```diff
@@ -248,6 +248,22 @@ Tab::Tab(BrowserWindow& window)
         update_status({}, count_waiting);
     };
 
+    view().on_restore_window = [this]() {
+        this->window().show();
+        this->window().move_to_front();
```

Comment:
omit `this`

Reply:
same below

Reply:
Can't - the compiler couldn't disambiguate between the `window()` method and a local variable named `window` further up in the file, even though only `this` is captured.